src/SonataCommentBundle.php

```diff
@@ -13,7 +13,6 @@
 
 namespace Sonata\CommentBundle;
 
-use Sonata\CoreBundle\Form\FormHelper;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
```

Comment:
CS check [is failing](https://github.com/sonata-project/SonataCommentBundle/pull/179/checks?check_run_id=539182671#step:4:14).